### ISSUE TITLE
Can't apply more than 10 tags to resource "aws_kinesis_stream"

### ISSUE THREAD

**Issue by ddpalmer** [terraform-providers/terraform-provider-aws#4562]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
Terraform v0.11.7
+ provider.aws v1.18.0

### Affected Resource(s)
aws_kinesis_stream

### Terraform Configuration Files

```hcl
provider "aws" {
  region  = "us-east-1"
  profile = "develop"
}

resource "aws_kinesis_stream" "kinesis_stream" {
  name             = "terraform_tags_test_stream"
  count            = 1
  shard_count      = 1
  retention_period = 24

  shard_level_metrics = [
    "IncomingBytes",
    "OutgoingRecords",
    "IteratorAgeMilliseconds",
    "IncomingRecords",
    "ReadProvisionedThroughputExceeded",
    "WriteProvisionedThroughputExceeded",
    "OutgoingBytes"
  ]

  tags {
    tag1  = "tag1"
    tag2  = "tag2"
    tag3  = "tag3"
    tag4  = "tag4"
    tag5  = "tag5"
    tag6  = "tag6"
    tag7  = "tag7"
    tag8  = "tag8"
    tag9  = "tag9"
    tag10  = "tag10"
    tag11  = "tag11"
  }
}
```

### Debug Output

https://gist.github.com/ddpalmer/1d403c3516b0de8c0f4abca8dc74144e

### Panic Output

None

### Expected Behavior

aws_kinesis_stream resource created with 11 tags.

### Actual Behavior

Tags create failed with error message: 'tags' failed to satisfy constraint: Member must have length less than or equal to 10

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

Nothing special

### References

Per AWS documentation kinesis streams may have up to 50 tags, however only 10 tags may be applied at a time. https://docs.aws.amazon.com/kinesis/latest/APIReference/API_AddTagsToStream.html

* #0000

**Comment by bflad** [comment]
Bug fix pull request submitted: #4574

**Comment by bflad** [comment]
The fix for this will release with v1.20.0 of the AWS provider, likely later today. 👍

**Comment by bflad** [comment]
This has been released in [version 1.20.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1200-may-23-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
